example.user-config.yml

```diff
@@ -21,6 +21,9 @@ pulp_install_plugins:
 # # Uncomment to deploy with apache instead of nginx
 # pulp_webserver_server: "apache"
 
+# # Comment to disable installer to enable repositories needed by Pulp
```

Comment:
Sounds like "you need to take responibility that the needed rpm repos are available". Should we write that down here?